package.json

```diff
@@ -17,7 +17,8 @@
     "stop:all": "npm stop; npm run stop:mockapi",
     "chimp": "NODE_ENV=acceptance chimp --mocha --browser=htmlunit --path=./test/acceptance/spec/",
     "chimp:dev": "NODE_ENV=acceptance chimp --mocha --path=./test/acceptance/ --watch --browser=firefox",
-    "lint": "eslint .",
+    "lint": "node_modules/.bin/eslint $FIXLINT .",
```

Comment:
Pretty sure, yes. There is some magic that happens on my system thanks to a special tweak of my user environment:
```bash
# Add `.bin` utilities from `node_modules`
export PATH=$PATH:./node_modules/.bin
```
...but this is something I had to setup and should not be assumed. The only executables we should assume are `node` and `npm`.